### DISCUSSION TITLE
我们为什么要“重新定义”开源术语 ？

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#29] [01 开源术语]
随着大家对开源价值的不断发掘与认可，开源已经从一个技术术语发展成一个大众词汇。而十四五规划将“开源”这个词的写入，更是吸引了更多的人关注。

我们在不同的场合，不同的应用场景下，讨论开源以及相关词汇的时候，很多时候都是鸡同鸭讲，并且由于“开源”系列词汇的宽泛性，很容易让人迷惑究竟大家聊的是否是一回事，因此我们需要一个讲述开源相关词汇的地方，特别是用中文来进行描述与解释。

以“开源”这个词为例，在很多正式场合，往往会追溯到“**开放源代码首创行动**”（Open Source Initiative，OSI）下面的“**开放源代码定义**”（Open Source Difinition，OSD），并认为只有符合 OSD 标准的代码才能称作开源代码。

而 OSI 则是由 “开放源代码首创行动组织”(Open Source Initiative Association，OSIA) 这个非营利民间组织所发起的，该组织就是由美国开源领域著名的 Bruce Perens 和 Erics. Raymond 等人于 1998 年所发起设立的，目的就是想尽可能统一开源软件的认证标准，捍卫开放源代码的定义，并因此成为开源软件认定、发布的权威组织。

OSIA 建立的意义之一在于：并非所有的公开了源代码的程序都可被称为“开放源代码”的程序， 只有符合OSD标准的程序才能使用“开放源代码软件”这一称谓。

在开放源代码首创行动组织的 OSD 中， 指明了该组织对开源软件及许可证的认定的标准，总共有 10 条，这里我们暂且不去详细罗列，后面有机会再来详细分析与解读。

OSIA 对开源软件许可证的认定程序是由许可证提供人启动的， 凡许可证提供人自己认为自己的许可证符合开源软件的要求的， 可以向 OSIA 提出加入 OSIA 开源软件许可证体系的申请， OSIA 将通过委员会审查和公众审查来决定是否承认该软件许可证为 OSI 系列的开源软件许可证。

中国首个开源协议**木兰**（Mulan）的“木兰宽松许可证”第二版（MulanPSL v2）也在 2020 年正式通过 OSI 认证，被正式批准为国际类别开源许可证（International licenses）。

近几年，开源发展从草根化逐渐向系统化、规模化方向过渡，业界对开源标准内涵和外延的认识已扩展，从最初的技术、框架、协议的业界共同遵守的事实标准，逐渐延伸至以开源治理、开发、应用、互联互通等范畴。与开源的内涵和外延相关的关键要素（开源规则、开源对象、参与主体、开源基础设施等）成为全数字产业的广泛共识性概念基础。

开源已在《十四五规划》中明确提出支持发展，但由于目前国内没有统一规范的定义规范，对“开源”、“开源社区”、“开源软件”、“开源硬件”、“开源治理”等基础性概念缺乏统一定义和认识，在政策落地执行过程中，各地方政府、不同行业、不同领域之间难以建立起有效的对话体系，亟需通过规范、甚至国家标准的方式，凝练产业共识，形成统一方法论和定义。

综上，我们可以看到目前国内在开源术语这块还存在的几个问题：

1. 在国际开源的大背景下，我们仍然需要开源相关术语的明确定义，即便是用中文将目前已经形成共识的英文术语描述一遍，这样方便建立一个统一的话语体系，并且提高大家对开源的认识；
2. 我们的开源术语规范，即要和国际上已有的开源规范或标准兼容，同时也需要考虑到中国国情，特别是在国际形式不确定的情况下，要能够为我国乃至世界进行持续服务，以更加开放的姿态行事；
3. 坚持创新方面的探索，今天开源的内涵不断变化，形成我们自己的创新点并让大家认可，也是需要从开源术语这样的基础工作做起，同时借机推动诸如知识产权、法律等其他领域的工作。

鉴于此，我们并没有”重新定义“开源术语，而是国际接轨的同时，逐渐过渡到以”我“为核心，持续推动全球开源生态的建设。

**Comment by zhuangbiaowei** [comment]
最近我写的一篇文章《[开源世界里的法律与政治](https://zhuangbiaowei.github.io/opensource/2022/03/07/law-and-politics-in-an-open-source-world.html)》，其中就提到了一个观点：

### 也许我们应该更进一步完善 Open Source Definition

> 所谓不歧视，应该不歧视任何使用，不歧视任何修改，不歧视任何分发，不歧视任何代码贡献，不歧视任何社区贡献！

而在原来的 OSD 中，可能会被错误的解读为：仅仅不歧视使用。

类似的“定义”，就是我希望做的事情——澄清，明确，细化。

**Reply by will-ww** [reply]
好咧，好好学习吸收下~
